packages/stylelint-config/index.js

```diff
@@ -1,184 +1,8 @@
-    'block-closing-brace-newline-after': 'always',
-    'block-closing-brace-newline-before': 'always-multi-line',
-    'block-closing-brace-space-before': 'always-single-line',
-    'block-opening-brace-newline-after': 'always-multi-line',
-    'block-opening-brace-space-after': 'always-single-line',
-    'block-opening-brace-space-before': 'always',
-    'comment-whitespace-inside': 'always',
-    'custom-property-empty-line-before': [
-      'always',
-      {
-        except: ['after-custom-property', 'first-nested'],
-        ignore: ['after-comment', 'inside-single-line-block'],
-      },
-    ],
-    'declaration-bang-space-after': 'never',
-    'declaration-bang-space-before': 'always',
-    'declaration-block-semicolon-newline-after': 'always-multi-line',
-    'declaration-block-semicolon-space-after': 'always-single-line',
-    'declaration-block-semicolon-space-before': 'never',
-    'declaration-block-single-line-max-declarations': 1,
-    'declaration-block-trailing-semicolon': 'always',
-    'declaration-colon-newline-after': 'always-multi-line',
-    'declaration-colon-space-before': 'never',
-    'declaration-empty-line-before': [
-      'always',
-      {
-        except: ['after-declaration', 'first-nested'],
-        ignore: ['after-comment', 'inside-single-line-block'],
-      },
-    ],
-    'function-comma-newline-after': 'always-multi-line',
-    'function-comma-space-before': 'never',
-    'function-max-empty-lines': 0,
-    'function-parentheses-newline-inside': 'always-multi-line',
-    'function-parentheses-space-inside': 'never-single-line',
-    'function-whitespace-after': 'always',
-    'media-feature-name-case': 'lower',
-    'media-feature-parentheses-space-inside': 'never',
-    'media-feature-range-operator-space-after': 'always',
-    'media-feature-range-operator-space-before': 'always',
-    'media-query-list-comma-newline-after': 'always-multi-line',
-    'media-query-list-comma-space-after': 'always-single-line',
-    'media-query-list-comma-space-before': 'never',
-    'no-eol-whitespace': [
-      true,
-      {
-        ignore: ['empty-lines'],
-      },
-    ],
-    'selector-attribute-brackets-space-inside': 'never',
-    'selector-attribute-operator-space-after': 'never',
-    'selector-attribute-operator-space-before': 'never',
-    'selector-combinator-space-after': 'always',
-    'selector-combinator-space-before': 'always',
-    'selector-descendant-combinator-no-non-space': true,
-    'selector-list-comma-space-before': 'never',
-    'selector-pseudo-class-case': 'lower',
-    'selector-pseudo-class-parentheses-space-inside': 'never',
-    'selector-pseudo-element-case': 'lower',
-    'selector-type-case': 'lower',
-    'value-list-comma-newline-after': 'always-multi-line',
-    'value-list-comma-space-after': 'always-single-line',
-    'value-list-comma-space-before': 'never',
-    'value-list-max-empty-lines': 0,
-  },
+  extends: [
+    './rules/stylelint-config-standard-scss',
```

Comment:
Great, much better. But it will have a greater sense if we differentiate those rules base on technology. For example all `scss/` rules should be in `scss.js` and the rest, which I think are base/general and applied to general CSS, in `css.js` or `base.js`.

I do not know if the rule is applied from recommended or standard config, but I know if it affects my SCSS or my CSS. So based on this I will go to a specific file.

This kind of sorting also allows us to add specific rules for instance `cssModules` or `order` or something else in the future.

Reply:
The thing is that Stylelint says that it doesn't know rules if the configs are extended just by `stylelint-config-standard`, we have to install and extend `stylelint-config-standard-scss` that extends `stylelint-config-standard` itself.

I just don't quiet get your differentiation technology drift/thought. As far as I understand how the process works, the Stylelint checks SCSS code and then Webpack pushes the SCSS to PostCSS which generates pure CSS. Why would you distinguish between CSS and SCSS if Stylelint always validates SCSS? 

Both packages (standard and recommended) validates SCSS, therefore the order of rules in this array is sorted to follow standards first, then check the higher level "recommended" rules and then some extras, CSS order properties etc.

I'm not sure if I'm answering your question here but I hope we'll finally meet at a clear point to manage the setup and your expectations.